src/mam4xx/nucleation.hpp

```diff
@@ -503,7 +503,7 @@ void newnuc_cluster_growth(Real ratenuclt_bb, Real cnum_h2so4, Real cnum_nh3,
     tmpa = max(tmpa, 0.0);
 
     // tmpb = h2so4 gas diffusivity ([m2/s], then [m2/h])
-    tmpb = 6.7037e-6 * pow(temp_in, 0.75) / cair;
+    tmpb = 6.7037e-9 * pow(temp_in, 0.75) / cair;
```

Comment:
What is this numerical factor? If it's going to absorb an additional `1e-3` to accommodate the omission above, should we separate this into that conversion factor and give a name to the `6.7e-6` portion?